### PR TITLE
Added Indonesian Translation

### PR DESCRIPTION
**Pull request by ykris45**
Hello My Name is Chris from The Republic of Indonesia. I have succeeded in translating the Decred Web into Indonesian as the national language of my country. I have worked as a software translator on various projects for 5 years. Thank You

### CODE REVIEW

**Comment by jholdstock** [comment]
Thanks for your submission @ykris45.

In order to coordinate a review, please consider joining [Matrix chat](https://chat.decred.org) where translation work is organized in a dedicated channel named `#translations:decred.org`

**Comment by ykris45** [comment]
> Thanks for your submission @ykris45.
> 
> In order to coordinate a review, please consider joining [Matrix chat](https://chat.decred.org) where translation work is organized in a dedicated channel named `#translations:decred.org`

you welcone sir, yes i just join on the decred matrix server yesterday

File: transifex_catalogs/id.json [hunk 1]
```diff
@@ -0,0 +1,103 @@
+{
+  "general_javascript_warning": "Diperlukan JavaScript.",
+  "general_javascript_warning_description": "Maaf, situs web ini tidak akan berfungsi dengan baik tanpa mengaktifkan JavaScript.",
+  "exchanges_title": "Bursa Perdagangan",
+  "exchanges_disclaimer": "Tidak ada bursa yang tercantum di atas selain <a target='_blank' rel='noopener noreferrer' href='https://dex.decred.org/'>DCRDEX</a> memungkinkan pengguna untuk menyimpan dana mereka selama proses pertukaran. Beberapa bursa terpusat termasuk jenis \"instan\" juga dapat meminta informasi pribadi untuk menyelesaikan perdagangan setelah dana disimpan. Pengguna sangat disarankan untuk berdagang di DCRDEX untuk memaksimalkan privasi mereka  <a target='_blank' rel='noopener noreferrer' href='https://www.youtube.com/watch?v=XaE_xzGKjeQ'>Tutorial DCRDEX</a> tersedia.",
+  "exchanges_description": "Pertukaran mata uang digital menyediakan cara mudah untuk membeli dan memperdagangkan Decred (DCR). ",
```
Comment (jholdstock) [inline transifex_catalogs/id.json:6]:
```suggestion
  "exchanges_description": "Pertukaran mata uang digital menyediakan cara mudah untuk membeli dan memperdagangkan Decred (DCR).",
```

File: transifex_catalogs/id.json [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  "title": "Decred — Uang Berkembang",
+  "description": "Decred adalah mata uang digital yang diatur secara otonom. ",
```
Comment (jholdstock) [inline transifex_catalogs/id.json:8]:
```suggestion
  "description": "Decred adalah mata uang digital yang diatur secara otonom.",
```

File: transifex_catalogs/id.json [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  "brand_guide_4": "Warna Primer",
+  "brand_guide_5": "Kunci biru",
+  "brand_guide_6": "Pirus",
+  "brand_guide_7": "Biru tua",
+  "brand_guide_8": "Tipografi",
+  "brand_guide_9": "Warna Pelengkap",
+  "brand_guide_10": "Biru muda",
+  "brand_guide_11": "Hijau",
+  "brand_guide_12": "Oranye",
+  "brand_guide_13": "Tujuan identitas visual Decred adalah untuk memastikan representasi unik di antara mata uang digital, mendukung penggunaan yang konsisten di berbagai platform serta mudah diingat dan menarik bagi pengguna baru. Hal ini dicapai melalui penggunaan logo, pengaturan jenis, arah warna, dan grafik yang benar. ",
```
Comment (jholdstock) [inline transifex_catalogs/id.json:36]:
```suggestion
  "brand_guide_13": "Tujuan identitas visual Decred adalah untuk memastikan representasi unik di antara mata uang digital, mendukung penggunaan yang konsisten di berbagai platform serta mudah diingat dan menarik bagi pengguna baru. Hal ini dicapai melalui penggunaan logo, pengaturan jenis, arah warna, dan grafik yang benar.",
```

File: transifex_catalogs/id.json [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+  "downloads_app_guarda": "Guarda diwakili oleh dompet multi-mata uang Web, Desktop, dan Seluler.",
+  "downloads_app_ledger": "Ledger Live adalah dompet berbasis web yang diamankan oleh dompet perangkat keras Ledger.",
+  "downloads_app_trezor": "Trezor adalah dompet perangkat keras yang didukung oleh dompet Decrediton dan Exodus.",
+  "downloads_app_trust": "Trust Wallet adalah dompet multi-koin yang aman untuk Android dan iOS.",
+  "stakepools_description_1": "Pemegang koin yang sudah dikurangi dapat ikut serta dalam tata kelola proyek dengan mengunci koin mereka secara acak dengan hadiah tiket untuk mendapatkan hadiah staking. Dengan berpartisipasi, pemangku kepentingan membantu mengamankan jaringan dengan memberikan suara untuk menyetujui pekerjaan para penambang, dan mendapatkan hak suara atas usulan  <a href='https://docs.decred.org/proof-of-stake/overview/' target='_blank' rel='noopener noreferrer'>dokumentasi</a>.",
+  "stakepools_description_2": "Karena tiket dipanggil secara pseudorandom dan dompet harus online untuk memilih ketika tiket dipanggil. banyak orang menggunakan Penyedia Layanan Voting (VSP) menggunakan Decrediton atau dcrwallet. ",
```
Comment (jholdstock) [inline transifex_catalogs/id.json:61]:
```suggestion
  "stakepools_description_2": "Karena tiket dipanggil secara pseudorandom dan dompet harus online untuk memilih ketika tiket dipanggil. banyak orang menggunakan Penyedia Layanan Voting (VSP) menggunakan Decrediton atau dcrwallet.",
```

File: transifex_catalogs/id.json [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+  "exchanges_category_decentralized": "Terdesentralisasi",
+  "exchanges_category_atomic_swaps": "Pertukaran Atomic",
+  "home_download_wallet": "Unduh Dompet",
+  "downloads_description": "Jika Anda baru mengenal Decred, pastikan untuk meninjau <a href='https://docs.decred.org' target='_blank' rel='noopener noreferrer'>dokumentasi</a>.  <a href='https://docs.decred.org/advanced/verifying-binaries/' target='_blank' rel='noopener noreferrer'>instruksi ini</a> untuk memverifikasi biner rangkaian aplikasi Decrediton dan Command-line.",
+  "downloads_category_core_software": "Perangkat lunak inti",
+  "downloads_category_third_party_wallets": "dompet pihak ketiga",
+  "downloads_category_hardware_wallets": "Dompet perangkat keras",
+  "downloads_app_decrediton": "Dompet UI grafis untuk Windows, macOS dan Linux.",
+  "downloads_app_dcrdex": "Dompet ringan multi-koin dengan dukungan DEX, Tanpa fitur staking atau dompet lanjutan.",
+  "downloads_app_cryptopower": "Dompet GUI ringan multi-koin untuk DCR, BTC, dan LTC. ",
```
Comment (jholdstock) [inline transifex_catalogs/id.json:51]:
```suggestion
  "downloads_app_cryptopower": "Dompet GUI ringan multi-koin untuk DCR, BTC, dan LTC.",
```

File: transifex_catalogs/id.json [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+  "exchanges_payment_title": "Pemroses Pembayaran yang Dikurangi",
+  "exchanges_category_instant_exchange": "Pertukaran Instan",
+  "exchanges_category_high_liquidity": "Likuiditas Tinggi",
+  "exchanges_category_fiat": "Mata Uang",
+  "exchanges_category_decentralized": "Terdesentralisasi",
+  "exchanges_category_atomic_swaps": "Pertukaran Atomic",
+  "home_download_wallet": "Unduh Dompet",
+  "downloads_description": "Jika Anda baru mengenal Decred, pastikan untuk meninjau <a href='https://docs.decred.org' target='_blank' rel='noopener noreferrer'>dokumentasi</a>.  <a href='https://docs.decred.org/advanced/verifying-binaries/' target='_blank' rel='noopener noreferrer'>instruksi ini</a> untuk memverifikasi biner rangkaian aplikasi Decrediton dan Command-line.",
+  "downloads_category_core_software": "Perangkat lunak inti",
+  "downloads_category_third_party_wallets": "dompet pihak ketiga",
```
Comment (jholdstock) [inline transifex_catalogs/id.json:47]:
```suggestion
  "downloads_category_third_party_wallets": "Dompet pihak ketiga",
```

File: transifex_catalogs/id.json [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+  "stakepools_warning_2": "Untuk mendorong desentralisasi jaringan, disarankan untuk menghindari penyedia yang memiliki lebih dari 5% suara jaringan.",
+  "news_title": "Berita",
+  "news_press_releases": "Siaran Pers",
+  "news_all": "Semua",
+  "news_coverage": "Cakupan",
+  "news_decred_journal": "Jurnal yang Diputuskan",
+  "news_software": "Rilis Perangkat Lunak",
+  "wallets_title": "Dompet",
+  "vsp_title": "Penyedia Layanan Pemungutan Suara",
+  "Page not found": "halaman tidak ditemukan",
```
Comment (jholdstock) [inline transifex_catalogs/id.json:75]:
```suggestion
  "Page not found": "Halaman tidak ditemukan",
```

**Review by jholdstock — COMMENTED** [review]
# Technical Review

All new strings are displayed OK on the site. All HTML elements embedded in the strings are proper. A few minor in-line suggestions made for extra whitespaces/missing caps. 

In order for Indonesian to be added to the language dropdown, please add the following segment to the bottom of `src/config.yml`:

```
    id:
        contentDir: content/id
        languageName: IND
        weight: 130
        params:
            languageDir: ltr
```

**Comment by jholdstock** [comment]
# Language review

I am not an Indonesian speaker, but I ran some quick sanity checks on the translated strings using Google Translate. I noticed some red flags.

Here are some of the strings which have been provided in this PR.

|Original|Translated in this PR|Reverse translated by me, with Google|
|---|---|---|
|Decred Developers|Pengembang yang Direduksi|Reduced Developer|
|Decred Journal|Jurnal yang Diputuskan|Decided Journal|
|Decred Governance|Tata Kelola yang Ditetapkan|Established Governance|
|Decred Payment Processors|Pemroses Pembayaran yang Dikurangi|Reduced Payment Processors|

Not only are these blatently incorrect, in every one of these cases you can paste the original English into Google Translate and you get a matching Indonesian result. I believe that the strings in this PR are the result of an automated translation and not the result of careful manual work,

**Comment by ykris45** [comment]
@jholdstock there are some cases of machine translation errors. for example discord, if you use google translate it will be a dispute. 
Decreed indeed in Indonesian can be translated as decided, it is based on https://helloenglish.com/translate/english-indonesian-dictionary/meaning-of-decreed-in-Indonesian
I'm not careful enough.

**Comment by ykris45** [comment]
I have corrected this. thank you for your kind input for me.

**Pull request closed without merging**